fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AdminStmtAnalyzer.java

```diff
@@ -0,0 +1,205 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.analyzer;
+
+import com.google.common.base.Strings;
+import com.starrocks.analysis.AdminSetConfigStmt;
+import com.starrocks.analysis.AdminSetReplicaStatusStmt;
+import com.starrocks.analysis.AdminShowConfigStmt;
+import com.starrocks.analysis.AdminShowReplicaDistributionStmt;
+import com.starrocks.analysis.AdminShowReplicaStatusStmt;
+import com.starrocks.analysis.BinaryPredicate;
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.SlotRef;
+import com.starrocks.analysis.StringLiteral;
+import com.starrocks.catalog.CatalogUtils;
+import com.starrocks.catalog.Replica;
+import com.starrocks.cluster.ClusterNamespace;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.sql.ast.AstVisitor;
+
+import java.util.List;
+import java.util.Map;
+
+public class AdminStmtAnalyzer {
+    public static void analyze(AdminSetConfigStmt adminSetConfigStmt, ConnectContext session) {
```

Comment:
Each statement should not implement an analyze statement, it should implement only one, the parameter can be StatementBase, and then the Visitor calls the corresponding visitXXX according to the specific type